package.json

```
@@ -3,16 +3,43 @@
   "private": true,
   "version": "3.3.3",
   "description": "🐻 Bear necessities for state management in React",
-  "main": "index.cjs.js",
-  "module": "index.js",
+  "main": "index.js",
+  "module": "index.module.js",
   "types": "index.d.ts",
   "files": [
     "**"
   ],
+  "exports": {
+    "./package.json": "./package.json",
+    ".": {
+      "types": "./index.d.ts",
+      "import": "./index.module.js",
```

The use of "import" may lead dual package hazard issue. We can try that road further but in #336 later.
For this PR, let's keep in sync with jotai/valtio.
```suggestion
      "module": "./index.module.js",
```
Would you change for all four?